### DISCUSSION TITLE
compatibility with torchmetrics >1

### DISCUSSION THREAD

**Discussion by Gornoka** [open-edge-platform/anomalib#1375] [Feature Requests] [answered]
Hi, 
are there any plans to increase the supported torchmetrics version to >1?
The last PR in that direction appears to be abandoned.
If it is not planned, would pr's in that direction be welcome?

**Comment by samet-akcay** [comment, marked as answer]
Hi @Gornoka, we are waiting for a pr to be merged on the torchmetrics side. Recent torchmetric AUC computation broke anomaly detection AUC computation. The torchmetrics team proposed a fix in a PR, which has not been merged yet.

You could refer to the details here
https://github.com/Lightning-AI/torchmetrics/pull/1676

**Comment by samet-akcay** [comment]
Closing this as it is currently being supported
https://github.com/openvinotoolkit/anomalib/blob/2705b9d3c532b91dfa9ac3ff42999685d734be61/pyproject.toml#L39

**Comment by Gornoka** [comment]
Thanks for addressing this issue @samet-akcay .
